docs/Command-Line-Interface.md

```diff
@@ -57,6 +57,36 @@ should have the final escaped parameters (cat /\\*).
     <td>Print the content of the file to the console.</td>
   </tr>
   <tr>
+    <td>chgrp</td>
+    <td>chgrp "group" "path"</td>
+    <td>Change the group of the directory or file.</td>
+  </tr>
+  <tr>
+    <td>chgrpr</td>
```

Comment:
instead of having this recursive version as a new command, can we make a `-R` option? I am suggesting `-R` is because I don't want to introduce too many commands with slightly difference function.